### ISSUE TITLE
Filter on binary data

### ISSUE THREAD

**Issue by moander** [googleapis/nodejs-bigtable#128]
I can write the buffer and retrieve it again using decode:false but I cannot figure out how to filter on the value.

```js
const buf = Buffer.from('a468c3a669', 'hex');

// Throws Can't convert to RegExp String from unknown type
{
   value: buf
}

// Returns zero rows instead of throwing
{
  value: [
    buf
  ]
}

// Using binary string also returns zero rows
{
   value: buf.toString('binary')
}
```

#### Environment details

  - `@google-cloud/bigtable` version: 0.13.1

**Comment by moander** [comment]
Filtering on binary values containing invalid utf-8 does not work. I have tested this on version 0.15.0.

```js
await bt.createTable('test5', {
    families: [
        'c'
    ]
});

await bt.table('test5').mutate([
    {
        key: 'a1',
        method: 'insert',
        data: { c: { test: Buffer.from('æ', 'binary') } }
    },
    {
        key: 'a2',
        method: 'insert',
        data: { c: { test: Buffer.from('x', 'binary') } }
    },
]);

let [rows] = await bt.table('test5').getRows({
    filter: [
        {
            value: [
                Buffer.from('æ', 'binary'),
                Buffer.from('x', 'binary'),
            ]
        }
    ]
});

console.log('got rows', rows.map(r => r.id));
```

Actual result
```
got rows [ 'a2' ]
```

Expected result
```
got rows [ 'a1', 'a2' ]
```

**Comment by sduskis** [comment]
@kolodny, is this still active?

**Comment by kinwa91** [comment]
@moander this should be addressed in #129. Please re-open if you've verified and it is still not working as expected.
